src/rhsmlib/services/register.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, cp):
 
     def register(self, org, activation_keys=None, environment=None, force=None, name=None, consumerid=None,
             **kwargs):
```

Comment:
@awood The command example from my first comment had wrong syntax (the force has to be set as `boolean` and not a `string`), but when I try to command with right syntax: `$ dbus-send --address="unix:abstract=/var/run/dbus-mUUQSAIRKQ,guid=ec769c55eff56e7e97f1683b599d53f1" --print-reply --dest='com.redhat.RHSM1.Register' '/com/redhat/RHSM1/Register' com.redhat.RHSM1.Register.Register string:"admin" string:"admin" string:"admin" dict:string:boolean:"force","true":string:string:"username","admin":string:string:"password","admin" dict:string:string:"",""` then I get error from previous command.

When system is already registered, then `username` and `password` are not set in `CPProvider` see: https://github.com/candlepin/subscription-manager/blob/4d70c8e93b79c44a9796b35495cd0aeab8b6ba80/src/rhsmlib/dbus/base_object.py#L92